### DISCUSSION TITLE
Can Webots use TCP or UDP connect to my raspiberry?

### DISCUSSION THREAD

**Discussion by meihe323** [cyberbotics/webots#5608] [General]
I wonder if webots can use TCP or UDP to transmit camera images from Robots's data or camera images,such as the sample robot mavic, to my raspiberry, and get some information back from my raspiberry?  I dont't use ROS, and I want to use simple net program to achieve it? Is there any APIs inside webots?

**Comment by omichel** [comment]
Please ask any technical question on https://robotics.stackexchange.com with the [webots](https://robotics.stackexchange.com/questions/tagged/webots) tag, so that we can answer you there.
